docs/dev/developer-guides/bridging/l1-l2.md

```diff
@@ -140,11 +146,12 @@ const gasPrice = await wallet.providerL1!.getGasPrice();
 // The calldata can be encoded the same way as for Ethereum
 const calldata = "0x...";
 const gasLimit = BigNumber.from(1000);
+const gasPerPubdataByte = BigNumber.from(80000);
```

Comment:
it is better to use 800, the same value as in SDK for L1->L2 transactions (if the value is too big, the user may actually overpay for the pubdata most of the time)